.github/workflows/release.yaml

```diff
@@ -0,0 +1,57 @@
+name: Build and Push Database Exporter
+
+on:
+  push:
+    tags:
+      - "*.*.*-*"
+
+env:
+  UBUNTU_VERSION: '22.04'
+  RUST_IMAGE_TAG: '1.73.0'
+  S3_BUCKET: s3://eur2ccd.concordium.com
+
+permissions:
+  id-token: write
+  contents: read
+
+jobs:
+  release-euro2ccd:
+    runs-on: ubuntu-latest
+    environment: release
+    steps:
+      - name: Get the eur2ccd version
+        env:
+          TAG: ${{ github.ref_name }}
+        run: echo "EUR2CCD_VERSION=${TAG%%-*}" >> $GITHUB_ENV
```

Comment:
You have to ensure that the tag correlates with Cargo.toml version for reference.

Look at misc tools for example for an example